core-utilities/src/main/java/edu/illinois/cs/cogcomp/core/datastructures/textannotation/View.java

```diff
@@ -409,6 +410,16 @@ private View createRestrictedView(ITransformer<View, Double> scoreTransformer) {
         return list;
     }
 
+    /**
+     * @return all the constituents that have exact span start/end.
+     */
+    public List<Constituent> getConstituentsWithSpan(IntPair span) {
+        List<Constituent> list = new ArrayList<>();
+        for(Constituent c : this.constituents) if (c.getSpan().equals(span)) list.add(c);
+        list.sort(TextAnnotationUtilities.constituentStartComparator);
```

Comment:
Minor confusion: If the spans are the same, sort should not have any effect right? Might be missing something here.